db/warehouse/migrate/20250818183500_add_hmis_table_configurations.rb

```diff
@@ -0,0 +1,23 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class AddHmisTableConfigurations < ActiveRecord::Migration[7.1]
+  def change
+    create_table :hmis_table_configurations do |t|
+      t.bigint :data_source_id, null: false # HMIS Data Source that configuration belongs to
```

Comment:
probably more standard to use `references`
```suggestion
      t.references :data_source, null: false # HMIS Data Source that configuration belongs to
```